content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -59,13 +59,16 @@ Use frontend data from RUM, as well as backend, infrastructure, and log informat
         clientToken: '<DATADOG_CLIENT_TOKEN>',
         ...otherConfig,
         service: "my-web-application",
-        allowedTracingOrigins: ["https://api.example.com", /https:\/\/.*\.my-api-domain\.com/, (origin) => origin === "https://api.example.com"]
+        allowedTracingUrls: ["https://api.example.com", /https:\/\/.*\.my-api-domain\.com/, (origin) => origin === "https://api.example.com"]
     })
     ```
 
     To connect RUM to Traces, you need to specify your browser application in the `service` field.
 
-    `allowedTracingOrigins` accepts JavaScript strings, regular expressions, and functions that match the origins called by your browser application, defined as: `<scheme> "://" <hostname> [ ":" <port> ]`.
+    `allowedTracingUrls` will match against the full URL (`<scheme>://<host>[:<port>]/<path>[?<query>][#<fragment>]`). It accepts the following types:
+      - `string`: evaluated using startsWith, so `https://api.example.com` will match `https://api.example.com/v1/resource`.
+      - `RegExp`: evaluated against the full URL
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** "against the full URL" is repeated twice, maybe this could be improved as well